src/collectors/files/files.py

```diff
@@ -11,7 +11,7 @@
 import os
 import re
 
-_RE = re.compile(r'([a-z0-9._-]+)[\s=:]([0-9-]+)(\.?\d*)')
+_RE = re.compile(r'([A-Za-z0-9._-]+)[\s=:]+(-?[0-9]+)(\.?\d*)')
```

Comment:
As noted, the files collector (which I also wrote) wouldn't accept an upper case metric name and/or a negative value.  This is fixes as well (see note above in initial commit):
Files Collector:
allows upper case metric names, and negative values